### PR TITLE
feat: make Instance#fetch force true by default, and fix force parameter

### PR DESCRIPTION
**Pull request by advaith1**
**Please describe the changes this PR makes and why it should be merged:**

Currently, (User|Channel|Message)#fetch ignores the `force` parameter, always treating it as false. This PR fixes it.

It also sets `force` to true on all instance fetch methods, after discussion in #library-discussions

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by iCrawl — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Comment by DTrombett** [comment]
While you're at it, could you fix the same problem with the other structures, like `Channel#fetch ()`?

File: src/structures/Channel.js [hunk 1]
```diff
@@ -105,8 +105,8 @@ class Channel extends Base {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/Channel.js:105]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

File: src/structures/DMChannel.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ class DMChannel extends Channel {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/DMChannel.js:66]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

File: src/structures/GuildBan.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ class GuildBan extends Base {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/GuildBan.js:55]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

File: src/structures/GuildMember.js [hunk 4]
```diff
@@ -304,7 +304,7 @@ class GuildMember extends Base {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/GuildMember.js:304]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

File: src/structures/Message.js [hunk 5]
```diff
@@ -724,8 +724,8 @@ class Message extends Base {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/Message.js:724]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

File: src/structures/User.js [hunk 6]
```diff
@@ -261,8 +261,8 @@ class User extends Base {
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
```
Comment (almeidx) [inline src/structures/User.js:261]:
```suggestion
   * @param {boolean} [force=true] Whether to skip the cache check and request the API
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by DTrombett — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by shinotheshino — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**